census/README.md

```diff
@@ -0,0 +1,50 @@
+# Agent Check: Census
+
+## Overview
+
+Monitoring offers critical insights into maintaining and reacting to development processes. Often, developers use a whole suite of tools and products, each monitored independently. To ease the monitoring process, Census integrates with Datadog to provide developers the ability to monitor their Census workflows, along with their internal monitoring, all within Datadog.
```

Comment:
Could you follow the outline below for your overview description

1. Describe Census in one to two sentences
2. How can customers take advantage of this integration? What benefit will this bring them?
3. What type of data is sent in/pulled out?